train.py

```diff
@@ -182,7 +182,7 @@ def evaluate_model(val_data_loaders, epoch = None, iteration = None, adapt_bn =
 	train_frontend = models.AugmentationFrontend(frontend, waveform_transform = make_transform(args.train_waveform_transform, args.train_waveform_transform_prob), feature_transform = make_transform(args.train_feature_transform, args.train_feature_transform_prob))
 	train_dataset = datasets.AudioTextDataset(args.train_data_path, labels, args.sample_rate, frontend = train_frontend if not args.frontend_in_model else None, min_duration = args.min_duration, max_duration = args.max_duration, time_padding_multiple = args.batch_time_padding_multiple)
 	train_dataset_name = '_'.join(map(os.path.basename, args.train_data_path))
-	sampler = datasets.BucketingBatchSampler(train_dataset, batch_size = args.train_batch_size, mixing = args.train_data_mixing, bucket = lambda example: int(math.ceil(((example['end'] - example['begin']) / args.window_stride + 1) / args.batch_time_padding_multiple))) #+1 mean bug fix with bucket sizing
+	sampler = datasets.BucketingBatchSampler(train_dataset, batch_size = args.train_batch_size, mixing = args.train_data_mixing, bucket = lambda example: int(math.ceil(((example[0]['end'] - example[0]['begin']) / args.window_stride + 1) / args.batch_time_padding_multiple))) #+1 mean bug fix with bucket sizing
```

Comment:
create a `transcripts.duration(t)` function and use it here and in other places where duration is calculated